testproject/Assets/Samples/PrefabPool/PrefabPoolOverrideExample.unity

```diff
@@ -852,7 +850,7 @@ GameObject:
   - component: {fileID: 1024114719}
   - component: {fileID: 1024114718}
   m_Layer: 0
-  m_Name: NetworkManager
+  m_Name: '[NetworkManager]'
```

Comment:
I had to change at least a thing in the scenes to cause Unity to re-serialize scenes down to disk.
So I surrounded `NetworkManager` objects in scenes with square brackets.